pom.xml

```diff
@@ -136,6 +136,12 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<!-- TODO: Why isn't the version taken from boot bom? -->
+			<dependency>
+				<groupId>org.springframework</groupId>
+				<artifactId>spring-core</artifactId>
+				<version>4.3.4.RELEASE</version>
```

Comment:
OK. It's because you explicitly depend on spring-core in spring-cloud-contract-verifier. I suspect it may be adequate to rely on the transitive dependency. I tried simply removing it and the build broke somewhere else, but it seems reasonable to simply remove it in both places to me.